### PR TITLE
added new blog post on creating custom tables

### PR DESCRIPTION
**Pull request by timbayer93**

### CODE REVIEW

File: content/posts/how-to-create-custom-tables/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    params:
+        description: "header pic"
+        showOnTop: true
+---
+
+# Introduction
+
+This tutorial will teach you how to create custom tables in Matplotlib, which are extremely flexible in terms of the design and layout. You’ll hopefully see that the code is very straightforward! In fact, the main methods we will be using are `ax.text()` and `ax.plot()`.
+
+I want to give a lot of credit to Todd Whitehead, [@CrumpledJumper](https://twitter.com/CrumpledJumper), who has created these types of tables for various Basketball teams and players. His approach to tables is nothing short of fantastic due to the simplicity in design and how he manages to effectively communicate data to his audience. I was very much inspired by his approach and wanted to be able to achieve something similar in Matplotlib.
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md]: Replace with `[Todd Whitehead](https://twitter.com/CrumpledJumper)`

File: content/posts/how-to-create-custom-tables/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+Before I begin with the tutorial, I wanted to go through the logic behind my approach as I think it's valuable and transferable to other  visualizations (and tools!).
+
+With that, I would like you to **think of tables as highly structured and organized scatterplots**. Let me explain why: For me, scatterplots are the most fundamental chart type (regardless of tool).
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md]: lower case after `:`

File: content/posts/how-to-create-custom-tables/index.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+![Scatterplots](scatterplots.png)
+
+For example ax.plot() automatically "connects the dots" to form a line chart or ax.bar() automatically "draws rectangles" across a set of coordinates. Very often (again regardless of tool) we may not always see this process happening. The point is, it is useful to think of any chart as a scatterplot or simply as a collection of shapes based on xy coordinates. This logic / thought process can unlock a ton of *custom* charts as the only thing you need are the coordinates (which can be mathematically computed).
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md]: Replace ax.plot() with `ax.plot()` (add ``)

Replace ax.bar() with `ax.bar()` (add ``)

File: content/posts/how-to-create-custom-tables/index.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+Importing required libraries.
+
+```python
+import matplotlib as mpl
+import matplotlib.patches as patches
+from matplotlib import pyplot as plt
+```
+
+First, we will need to set up a coordinate space - I like two approaches;
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md]: Replace ; with :

File: content/posts/how-to-create-custom-tables/index.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+newaxes[0].plot([0, 1, 2, 3], [1, 2, 0, 2], c='black')
+newaxes[0].set_ylim(-1, 3)
+
+# once again, the key is to hide the axis!
+newaxes[0].axis('off')
+```
+
+![Sparklines](8_sparklines.png)
+
+That’s it, custom tables in Matplotlib. I did promise very simple code and an ultra-flexible design in terms of what you want / need. You can adjust sizes, colors and pretty much anything with this approach and all you need is simply a loop that plots text in a structured and organized manner. I hope you found it useful.
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md]: Can you add a link to the full code / jupyter notebook for the reader to download?

File: content/posts/how-to-create-custom-tables/index.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+Adding a title is also straightforward.
+
+```python
+ax.set_title(
+	'A title for our table!',
+	loc='left',
+	fontsize=18,
+	weight='bold'
+)
+```
```
Comment (davidevaleriani) [inline content/posts/how-to-create-custom-tables/index.md:194]: Can you add another image showing the title?

**Review by davidevaleriani — COMMENTED** [review]
Great article, thank you Tim! I have added a few minor comments to make it easier to read.
After your revisions, I'll merge. Thanks!

**Comment by timbayer93** [comment]
Changes made, thanks Davide.

**Pull request merged**